.github/workflows/handler-test.yml

```diff
@@ -213,6 +213,15 @@ jobs:
           TFE_TOKEN: ${{ secrets.TFE_TOKEN }}
         run: terraform destroy -auto-approve -input=false -no-color
 
+      - name: Terraform Delete Workspace
+        id: destroy
```

Comment:
Should the `id` be unique? Maybe `destroy-workspace`?

Reply:
Added